go/test/endtoend/cellalias/cell_alias_test.go

```diff
@@ -63,7 +63,6 @@ var (
 		"--enable_replication_reporter",
 		"--serving_state_grace_period", "1s",
 		"--binlog_player_protocol", "grpc",
-		"--enable-autocommit",
```

Comment:
I don't know why these tests customize the vttablet args :(
They have been that way from the beginning.